docs/SUMMARY.md

```diff
@@ -7,6 +7,7 @@
 * [Guidelines](guidelines/README.md)
     * [Style guide outline](guidelines/outline.md)
     * [Example style guide](guidelines/example_en-us.md)
+    * [Marketing guide to Firefox localization](guidelines/marketing_guide_to_firefox_l10n.md)
```

Comment:
This needs to be shorter, I don't think it'll fit.

I just realized that you're working directly from GitHub, ideally this should be in one PR, with both documents and the summary changes.